web/app/models/DrivingSchool.js

```diff
@@ -0,0 +1,16 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving school model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSchoolSchema = new mongoose.Schema({
+    schoolId: { type: Number, required: true },
+    owners: [{ type: Schema.Types.ObjectId, ref: "User" }],
+    instructors: [{ type: Schema.Types.ObjectId, ref: "User" }],
+    students: [{ type: Schema.Types.ObjectId, ref: "User" }],
+    address: { type: String, required: true },
```

Comment:
Two lines for an address does make sense. Is there somewhere in the web UI where we will be displaying the address for a school? If so, then the storage of an address can be changed.